source/Calamari.Tests/KubernetesFixtures/ApiResourceOutputParserTests.cs

```diff
@@ -0,0 +1,49 @@
+"podtemplates                                     v1                                true         PodTemplate                        create,delete,deletecollection,get,list,patch,update,watch   ",
+"replicationcontrollers              rc           v1                                true         ReplicationController              create,delete,deletecollection,get,list,patch,update,watch   all",
+"resourcequotas                      quota        v1                                true         ResourceQuota                      create,delete,deletecollection,get,list,patch,update,watch   ",
+"secrets                                          v1                                true         Secret                             create,delete,deletecollection,get,list,patch,update,watch   ",
+"serviceaccounts                     sa           v1                                true         ServiceAccount                     create,delete,deletecollection,get,list,patch,update,watch   ",
+"services                            svc          v1                                true         Service                            create,delete,deletecollection,get,list,patch,update,watch   all",
+"mutatingwebhookconfigurations                    admissionregistration.k8s.io/v1   false        MutatingWebhookConfiguration       create,delete,deletecollection,get,list,patch,update,watch   api-extensions",
+"validatingadmissionpolicies                      admissionregistration.k8s.io/v1   false        ValidatingAdmissionPolicy          create,delete,deletecollection,get,list,patch,update,watch   api-extensions",
+"validatingadmissionpolicybindings                admissionregistration.k8s.io/v1   false        ValidatingAdmissionPolicyBinding   create,delete,deletecollection,get,list,patch,update,watch   api-extensions",
+"validatingwebhookconfigurations                  admissionregistration.k8s.io/v1   false        ValidatingWebhookConfiguration     create,delete,deletecollection,get,list,patch,update,watch   api-extensions",
+"customresourcedefinitions           crd,crds     apiextensions.k8s.io/v1           false        CustomResourceDefinition           create,delete,deletecollection,get,list,patch,update,watch   api-extensions",
+"apiservices                                      apiregistration.k8s.io/v1         false        APIService                         create,delete,deletecollection,get,list,patch,update,watch   api-extensions",
+"controllerrevisions                              apps/v1                           true         ControllerRevision                 create,delete,deletecollection,get,list,patch,update,watch   "
+            };
+
+            var parsedResult = ApiResourceOutputParser.ParseKubectlApiResourceOutput(outputLines);
+
+            parsedResult.Should()
+                        .NotBeEmpty();
```

Comment:
Would be worth asserting that the output is as expected - I'd be happy with a really cut down list compared to what you've got to save the typing.

Reply:
ah yeah, I meant to do that. I think the original reason it was like this was I was using the test to debug and fix it... Good catch!

Reply:
Actually tested in [30b70a4](https://github.com/OctopusDeploy/Calamari/pull/1423/commits/30b70a47b90731774233af3c34ff789ba46b7d18)